<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>480</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>32</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="selectPluginBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>210</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>Select CAN plugin</string>
   </property>
   <widget class="QComboBox" name="pluginListBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="specifyInterfaceNameBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>210</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>Specify CAN interface name</string>
   </property>
   <widget class="QComboBox" name="interfaceListBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="interfacePropertiesBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>210</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>CAN interface properties</string>
   </property>
   <widget class="QLabel" name="descriptionLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Interface:</string>
    </property>
   </widget>
   <widget class="QLabel" name="serialNumberLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial:</string>
    </property>
   </widget>
   <widget class="QLabel" name="channelLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Channel:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isFlexibleDataRateCapable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Flexible Data Rate</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="isVirtual">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>129</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Virtual</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="configurationBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>260</width>
     <height>300</height>
    </rect>
   </property>
   <property name="title">
    <string>Specify configuration</string>
   </property>
   <widget class="QLabel" name="rawFilterLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RAW Filter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorFilterEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>FrameError bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorFilterLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Error Filter</string>
    </property>
   </widget>
   <widget class="QComboBox" name="loopbackListBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="receiveOwnListBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="BitRateBox" name="bitRateListBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="canFdListBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="BitRateBox" name="dataBitRateListBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="loopbackLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Loopback</string>
    </property>
   </widget>
   <widget class="QLabel" name="receiveOwnLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Receive Own</string>
    </property>
   </widget>
   <widget class="QLabel" name="bitRateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bit Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataBitRateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Bit Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="canFdLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CAN FD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rawFilterListBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="useConfigurationBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Configuration</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BitRateBox</class>
   <extends>QComboBox</extends>
   <header>src/main/bitrate_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
